### PR TITLE
ADD: fixes and navbar for user

### PR DESCRIPTION
**Pull request by Camilo-J**

### CODE REVIEW

**Comment by Camilo-J** [comment]
Good Job!!

File: src/AuthenticatedApp.jsx [hunk 1]
```diff
@@ -76,6 +77,16 @@ function AuthenticatedApp() {
...
      
       {user ? (
+
+        <>
+        <Navbar_professional />
+        <Routes>
+            <Route path={"/home"} element={<Search />} />
+            <Route path={"/"} element={<Search />} />
+          </Routes>
+        </>
```
Comment (paulotijero) [inline src/AuthenticatedApp.jsx:87]: You can also use `Outlet` to build a layout that includes the `navbar` , just it a suggestion [Routing Docs - Outlet](https://github.com/remix-run/react-router/blob/f59ee5488bc343cf3c957b7e0cc395ef5eb572d2/docs/installation/getting-started.md#relative-links)

File: src/components/navbar.jsx [hunk 2]
```diff
@@ -99,29 +100,29 @@ function Navbar() {
...
           </Authors>
 
           <Authors>
             <Github>
-              <RiGithubFill />
+            <a href="https://github.com/Camilo-J" target="_blank"><RiGithubFill /></a>
             </Github>
-            <p>Camilo Huanca</p>
+            <a href="https://github.com/eliasmaq" target="_blank"><p>Camilo Huanca</p></a>
           </Authors>
```
Comment (paulotijero) [inline src/components/navbar.jsx:126]: Exists a vulnerability relate to `target="_blanl"` a would-be solution is adding `rel="noopener noreferrer"` , see more in this [link to stackoverflow](https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable)

**Review by paulotijero — COMMENTED** [review]
Great job, leave some suggestions 👀

**Pull request merged**